README.md

```diff
@@ -33,13 +33,14 @@ Flags:
   -o, --operator-namespaces strings    Comma-separated list of namespace(s) in which operator(s) are running (default [elastic-system])
       --output-directory string        Path where to output diagnostic results
   -r, --resources-namespaces strings   Comma-separated list of namespace(s) in which resources are managed
+      --run-agent-diagnostics          Run diagnostics on deployed Elastic Agents. Warning: credentials will not be redacted and appear as plain text in the archive
```

Comment:
Do we want to consider trying to fix this issue in a follow-up? As we (ECK) control the configuration of the agent, so we should be able to sanitize at least the api_key?

Reply:
https://github.com/elastic/eck-diagnostics/issues/99